### ISSUE TITLE
Error "unexpected EOF" when connecting to WebSocket endpoint

### ISSUE THREAD

**Issue by miguelmota** [trufflesuite/ganache-core#63]
Trying to connect to websocket endpoint but get an unexpected error

## Current Behavior
`unexpected EOF`

## Steps to Reproduce (for bugs)

```js
var Ganache = require('ganache-core')
var server = Ganache.server({
  ws: true,
  port: 8545
})
var port = 8545
server.listen(port, (err) => {
  console.log(`Listening on ${port}`)
})
```

## Context
Trying to listen to events

## Your Environment
* Version used:  `2.1.0-beta.0`
* Environment name and version: node `v7.9.0`
* Operating System and version: `Darwin computer.local 16.7.0 Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64 x86_64`

**Comment by seesemichaelj** [comment]
This got fixed somewhere in the midst. Please update to the latest version. Thanks!
